runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotInput.java

```diff
@@ -58,6 +66,9 @@ public void keyReleased(KeyEvent event)
 		if (!config.isScreenshotEnabled())
 			return;
 
+		if (event.getKeyCode() == KeyEvent.VK_CONTROL)
+			this.CONTROL_KEY_DOWN = false;
+
 		if (event.getKeyCode() == KeyEvent.VK_INSERT)
 		{
 			plugin.takeScreenshot(TIME_FORMAT.format(new Date()));
```

Comment:
Is this on purpose? Insert is still bound to take a screenshot on release.

Reply:
@Tobi1202 keeping the insert keybind was purposeful, as I didn't want to remove functionality that many people might be used to.

However, pointing this out made me realize that the CONTROL_KEY_DOWN flag will not be set to false unless a user has the insert keybind option set to true, so I've fixed that!